lms/djangoapps/verify_student/management/commands/tests/test_manual_verify_student.py

```diff
@@ -0,0 +1,55 @@
+"""
+Tests for django admin commands in the verify_student module
+
+"""
+
+from django.core.management import call_command
+from django.test import TestCase
+from nose.tools import assert_equals
+from lms.djangoapps.verify_student.models import ManualVerification
+from lms.djangoapps.verify_student.utils import earliest_allowed_verification_date
+from student.tests.factories import UserFactory
+
+
+class TestVerifyStudentCommand(TestCase):
+    """
+    Tests for django admin commands in the verify_student module
+    """
+
+    def setUp(self):
+        super(TestVerifyStudentCommand, self).setUp()
+        self.user1 = UserFactory.create()
+        self.user2 = UserFactory.create()
```

Comment:
looking at the tests I think we only need on user `self.user = UserFactory.create()`

Reply:
both users are being used in the 'test_manual_verifications' test

Reply:
But it seems that each test below only need one user individually.

Reply:
@zubair-arbi The `test_manual_verifications` test method seems to be [using both `user1` and `user2`](https://github.com/edx/edx-platform/pull/18337/files#diff-b07dbeda49a49be15668e8511a45906dR29). I think it makes sense to keep both in there to verify that it handles multiple user emails.